examples/access-controllable-examples/README.md

```diff
@@ -0,0 +1,274 @@
+# Example of using the Access Control plugin
+
+An access control mechanism that allows you to specify which groups of users can access certain functions.
+
+```rust
+use near_plugins::AccessControllable;
+use near_plugins::AccessControlRole;
+use near_plugins_derive::access_control;
+use near_plugins_derive::access_control_any;
+use near_sdk::near_bindgen;
+use borsh::{BorshSerialize, BorshDeserialize};
+use near_plugins::events::AsEvent;
+use near_sdk::env;
+
+/// All types of access groups
+#[derive(AccessControlRole, Clone, Copy)]
+pub enum UsersGroups {
+    GroupA,
+    GroupB,
+}
+
+#[near_bindgen]
+#[access_control(role_type="UsersGroups")]
+#[derive(Default, BorshSerialize, BorshDeserialize)]
+struct Counter {
+    counter: u64,
+}
+
+#[near_bindgen]
+impl Counter {
+    /// In the constructor we set up a super admin, 
+    /// which can control the member lists of all user groups
+    #[init]
+    pub fn new() -> Self {
+        let mut contract: Counter = Self{
+            counter: 0,
+            __acl: __Acl::default(),
+        };
+
+        contract.__acl.init_super_admin(&near_sdk::env::predecessor_account_id());
+
+        contract
+    }
+
+    /// unprotected function, every one can call this function
+    pub fn unprotected(&mut self) {
+        self.counter += 1;
+    }
+
+    /// only the users from GroupA can call this method
+    #[access_control_any(roles(UsersGroups::GroupA))]
+    pub fn level_a_incr(&mut self) {
+        self.counter += 1;
+    }
+
+    /// only the users from GroupA or GroupB can call this method
+    #[access_control_any(roles(UsersGroups::GroupA, UsersGroups::GroupB))]
+    pub fn level_ab_incr(&mut self) {
+        self.counter += 1;
+    }
+
+
+    /// view method for get current counter value, every one can use it
+    pub fn get_counter(&self) -> u64 {
+        self.counter
+    }
+}
+```
+
+## The contract methods description
+### acl_storage_prefix
+`acl_storage_prefix` is a method that returns the common prefix of keys for storing the members and the admins of groups.
+`__acl` by default.
+
+```shell
+$ near call <CONTRACT_ACCOUNT> acl_storage_prefix --accountId <ALICE_ACCOUNT>
+Scheduling a call: <CONTRACT_ACCOUNT>.acl_storage_prefix()
+Doing account.functionCall()
+Transaction Id <TRANSACTION_ID>
+To see the transaction in the transaction explorer, please open this url in your browser
+https://explorer.testnet.near.org/transactions/<TRANSACTION_ID>
+[ 95, 95, 97, 99, 108 ]
+$ python3
+>>> print(' '.join(str(b) for b in bytes("__acl", 'utf8')))
+95 95 97 99 108
+```
+
```

Comment:
fixed: https://github.com/aurora-is-near/near-plugins/pull/34/commits/6ce74158b38e3243b03abbe8b25d827601c1547b

Reply:
@mooori is it possible to remove the super admin somehow?

Reply:
There is [`revoke_super_admin_unchecked`](https://github.com/aurora-is-near/near-plugins/blob/5579de43c28984d00fa9aebc0a8e755258bba6c2/near-plugins-derive/src/access_controllable.rs#L169-L171), it can be called like this from inside a contract method:

```rust
self.__acl.revoke_super_admin_unchecked(&account_id);
```